core/src/main/scala/org/apache/spark/internal/config/UI.scala

```diff
@@ -129,6 +129,21 @@ private[spark] object UI {
     .intConf
     .createWithDefault(1000)
 
+  val UI_TIMELINE_JOBS_MAXIMUM = ConfigBuilder("spark.ui.timeline.jobs.maximum")
```

Comment:
Shall we write down these new configurations into `configuration.md`?

Reply:
I've added descriptions about the new properties and the existing `spark.ui.timeline.tasks.maximum`.